src/mqt/benchviewer/main.py

```diff
@@ -53,21 +53,20 @@ def __init__(
 
 @app.route(f"{PREFIX}/", methods=["POST", "GET"])
 @app.route(f"{PREFIX}/index", methods=["POST", "GET"])
-def index():
+def index() -> Any:
     """Return the index.html file together with the benchmarks and nonscalable benchmarks."""
-
     return render_template(
         "index.html",
-        benchmarks=SERVER.backend.benchmarks,
-        nonscalable_benchmarks=SERVER.backend.nonscalable_benchmarks,
+        benchmarks=SERVER.backend.benchmarks,  # type: ignore[attr-defined]
+        nonscalable_benchmarks=SERVER.backend.nonscalable_benchmarks,  # type: ignore[attr-defined]
     )
 
 
 @app.route(f"{PREFIX}/get_pre_gen", methods=["POST", "GET"])
-def download_pre_gen_zip():
+def download_pre_gen_zip() -> Any:
     filename = "MQTBench_all.zip"
 
-    if SERVER.activate_logging:
+    if SERVER.activate_logging:  # type: ignore[attr-defined]
```

Comment:
These ignores are necessary because "SERVER' is a global variable (the only one remaining after the ruff update marking all of them, which was the main reason for the other PR). The variable is initialized after the creation. Therefore, mypy marks them as possibly having a "None" value.

Reply:
could this be mitigated by something like an `assert SERVER is not None` or some kind of assert that it's an instance of a class or a cast or by typing the global variable?

Reply:
I typed now the global variable because I prefer that over having many assert statements which lead to new `unreachable` errors since mypy is assuming that `SERVER==None`. However, I had to add an `# type: ignore[assignment]` to the variable declaration. Still ugly, but the better ways for now in my opinion.